### PR TITLE
Final main fields

### PR DESCRIPTION
**Pull request by siegfault**
I'd love some input especially re: the last commit message about styling. Here's how it looks:

image

image

### CODE REVIEW

**Comment by mbenfowler** [comment]
I barely used `grid` styling at Turing so can't offer too much input.  I've also just been running into the same thing on the monster sheet, and have likewise just massaging things into place as needed.  It looks like a lot of elements have the `relative` and `bottom` properties, which would cause a lot of cascading squnching messing with margin stuff.

File: src/actor/character/sheets-tabs/main-tab.hbs [hunk 1]
```diff
@@ -1,10 +1,31 @@
 <div class="main-tab">
   <div class="left-content">
+    <div class="initiative-modifier">
+      <h4>
+        {{localize "BIO.INITIATIVE_MODIFIER"}}
+        <input id="number-input" type="number" name="system.baseAttribute.initiative_modifier.value" value="{{this.actor.attributes.initiative_modifier.value}}" />
```
Comment (AllPurposeName) [inline src/actor/character/sheets-tabs/main-tab.hbs:6]: Mind switching number-input to a class? I don't remember why I made it an id, but I'm guessing it was a mistake.

Comment (siegfault) [reply src/actor/character/sheets-tabs/main-tab.hbs:6]: Mind if I do that in a followup? I'm now noticing that the other id (`calculated-attribute-value`) should also be a class

**Review by mbenfowler — APPROVED** [review]

File: src/actor/character/sheets-tabs/main-tab.hbs (hunk 1 above)
Comment (siegfault) [reply src/actor/character/sheets-tabs/main-tab.hbs:6]: Addressed here: https://github.com/AllPurposeName/fabula-ultima-foundry-vtt/pull/49

**Pull request merged**